django/db/migrations/questioner.py

```diff
@@ -287,6 +305,12 @@ def ask_unique_callable_default_addition(self, field_name, model_name):
                 sys.exit(3)
         return None
 
+    def ask_db_table_add(self, model_name, db_table):
```

Comment:
Forgot to add this earlier... 
This method does not actually ask the user anything, so a name like `quit_for_db_table_add()` may be more appropriate. It will also make it clearer, where the method is called, that it is not expected to return.

Also, please check if a `return None` implementation needs to be added to the parent, non-interactive questioner class; other `ask_...` methods in this class do have such parallels.